src/main/java/Ui/Ui.java

```diff
@@ -3,42 +3,65 @@
 import java.util.List;
 import Task.*;
```

Comment:
perhaps it's better not to import entire package using *? as it violates the coding standard